test/deserialize/record.js

```diff
@@ -26,6 +26,29 @@ it('should deserialize a record of a known type', (test) => {
 })
 
 
+it('should deserialize a record class of a known type', (test) => {
```

Comment:
"deserialize a record of a known class type"

Je to detail, ale "record class" je jako prototyp recordu, ne konkrétní record. Upravim to sám.